app/public/css/styles.css

```diff
@@ -553,6 +562,8 @@ button[data-balloon] {
 
 .alert-content {
   text-align: left;
+  max-height: 180px;
+  overflow: hidden;
 }
```

Comment:
did you consult the overflow property with someone from design? I don't think its a good idea to hide scrollbars and just cut the content, but I might be wrong

Reply:
I did not consult this with anyone from design, but we can discuss this on team meeting, I added the `max-height` and `overflow: hidden` just because of the unexpectedErrorModal in which I am displaying part of the json that is about to be send to sentry. But I dont really see a problem with the hidden overflow. First of all, there is no other case in which this gets applied other than unexpectedErrorModal since we simply dont insert that much content into the alert, secondly IMO the modal looked a lot worse with two scrollbars but I am open to other opinions.